float32 voltage5V_v		# peripheral 5V rail voltage
uint8 usb_connected		# USB is connected when 1
uint8 brick_valid		# brick power is good when 1
uint8 servo_valid		# servo power is good when 1
uint8 periph_5V_OC		# peripheral overcurrent when 1
uint8 hipower_5V_OC		# hi power peripheral overcurrent when 1
